enterprise/frontend/src/metabase-enterprise/notifications/components/RequestNotificationsApp.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { useEffect, useState } from "react";
+import PropTypes from "prop-types";
+import { withRouter } from "react-router";
+import _ from "underscore";
+import { t } from "ttag";
+
+import Questions from "metabase/entities/questions";
+import User from "metabase/entities/users";
+
+import {
+  getRequestStatuses,
+  buildModerationRequestPath,
+} from "metabase-enterprise/moderation";
+
+import NotificationSectionHeader from "./NotificationSectionHeader";
+import RequestNotificationCard from "./RequestNotificationCard";
+import { ListContainer } from "./RequestNotificationsApp.styled";
+import { ModerationRequestApi } from "metabase/services";
+RequestNotifications.propTypes = {
+  questions: PropTypes.array,
+  users: PropTypes.array,
+  router: PropTypes.object.isRequired,
+};
+
+function RequestNotifications({ questions, users, router }) {
+  const [requests, setRequests] = useState([]);
+
+  const includedStatuses = getIncludedStatuses(router.location.query);
+  const usersById = _.indexBy(users, "id");
+  const questionsById = _.indexBy(questions, question => question.id);
+
+  useEffect(() => {
+    ModerationRequestApi.get().then(requests => {
```

Comment:
Is this temporary here? It can throw an error if a user changes the page before the request is resolved to that `setRequests` will be called on an unmounted component

Reply:
Yep this is temporary. BE will be implementing the real BE endpoints soon and I'll fix this up then